roombasim/pittras/task/land_task.py

```diff
@@ -0,0 +1,36 @@
+
+import numpy as np
+
+import roombasim.config as cfg
+
+from roombasim.ai import Task, TaskState
+
+class LandTaskState(object):
+    init = 0
+    descend = 1
+    done = 2
+
+class LandTask(Task):
+
+    def __init__(self):
+        # state machine
+        self._state = LandTaskState.init
+
+    def update(self, delta, elapsed, state_controller, environment):
+        drone_state = state_controller.query('DroneState', environment)
+
+        height = drone_state['z_pos']
+
+        # Check if we reached the target landing height tolerance
+        if height > 0:
```

Comment:
this is fine for now since position will be set to zero if the drone goes below the ground. However, in the future we might implement some leg sensors that could give us a yes/no reading for if we are on the ground

Reply:
Would this be blocking on this change or should this be added to some planned changes file going forward?

Reply:
fine for now